content/how-to/rackspace-fabric/Configure MFA/index.md

```diff
@@ -0,0 +1,73 @@
+---
+permalink: "Multi-Factor Authentication"
```

Comment:
no quotations needed and don't need spaces for permalinks
should just be `multi-factor-authentication` or `multifactor-authentication`. I assume the first may be more proper right?